### DISCUSSION TITLE
Plugdata on macOS 13.2?

### DISCUSSION THREAD

**Discussion by sevdh** [plugdata-team/plugdata#483] [Q&A] [answered]
I am following a discussion on the PD list and saw that the current Puredata vanilla has audio problems on macOS 13.2 because of a too old version of Portaudio being included. A test version that resolves the issue has been provided by Dan Wilcox.
I'm curious if Plugdata might have the same problem of running on macOS 13.2 or not. The reason I'm asking is that I would like to inform students about Plugdata and some of them might be running 13.2.

**Comment by timothyschoen** [comment, marked as answer]
Hi Edwin!

I'm running macOS 13.0 over here, so I'll update soon to test this. I think JUCE talks to the CoreAudio API directly, so I expect that there are no problems like that in plugdata!

**Comment by sevdh** [comment]
Hi TIm!

Thanks for the quick answer! I see, no Portaudio involved in Plugdata :-)
